server.js

```diff
@@ -17,6 +25,31 @@ app.get('/', function (req, res) {
     res.send('Welcome');
   });
 
+
+
+// connect to the MongoDB with mongoose
+require('./config/database');
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7812747)
```suggestion
require("./config/database");
```